docs/writing-rules/testing-rules.md

```diff
@@ -211,7 +211,7 @@ The semgrep rules are pulled from `p/semgrep-rule-lints`.
 
 This feature is still experimental and under active development. Your feedback is welcomed!
 
-## Enabling autofix in Semgrep App
+## Enabling autofix in Semgrep Cloud Platform
```

Comment:
Should this be for Semgrep Code only?

Reply:
I initially thought that some autofix could (technically) be applied to SSC, but you're right, it's primarily SAST (Code).